CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing to data-multi-subject
+
+To submit changes -- new images and/or manual ground truths (e.g., spinal cord segmentations, disc labels, etc.) -- please fork this repository and submit a pull request.
+The process is a little bit unusual because of the existence of `git-annex` and Amazon S3 in the mix:
+
+## If you are a member of the lab
+
+1. Fork this repository.
+2. Add your fork as a remote:
+```
+# in your local copy, assuming you already followed the download instructions in README.md
+git add yourusername git@github.com:yourusername/data-multi-subject.git
+git annex sync yourusername
+```
+3. Start a branch:
+```
+git checkout -b your-changes
+```
+4. Make your changes. Use the usual git commands (`git add -u`, `git add [filename]`, `git rm [filename]`, etc) to apply them.
+5. Commit them as usual to git:
+```
+git commit
+```
+6. Contact [@jcohenadad](https://github.com/jcohenadad) to get Amazon S3 credentials to the cloud storage used for the NifTi files.
+```
+export AWS_ACCESS_KEY_ID="..." AWS_SECRET_ACCESS_KEY="..."
+```
+7. Publish:
+```
+git annex sync --content amazon
+git annex sync yourusername
+```
+8. Go to https://github.com/yourusername/data-multi-subject and click the Pull Request button.
```

Comment:
@kousu i see these 3 PR options:

![image](https://user-images.githubusercontent.com/2482071/90458653-fcf3ae80-e0cc-11ea-8d9b-5375f9fc65fb.png)

should it be "jca/participants"?

Reply:
Yes. `git-annex` and github aren't really designed to work together I guess!